servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -259,6 +252,7 @@ def cromwell_query_params(query, page, page_size):
     query_params.append({'page': str(page)})
     query_params.append({'additionalQueryResultFields': 'parentWorkflowId'})
     query_params.append({'additionalQueryResultFields': 'labels'})
+    query_params.append({'includeSubworkflows': 'false'})
```

Comment:
It is very likely we won't hit the issue, since Job Manager is using pagination to query Cromwell./ Just want to comment that there is an issue with the current version of Cromwell if you use `includeSubworkflows` to query for more than ~1000 workflows: https://github.com/broadinstitute/cromwell/issues/3873